Monika After Story/game/script-ch30.rpy

```diff
@@ -492,15 +492,21 @@ label ch30_autoload:
 
     # check persistent to see if player put Monika to sleep correctly
     elif persistent.closed_self:
-        python:
 
-            sel_greeting_event = store.mas_greetings.selectGreeting()
-            selected_greeting = sel_greeting_event.eventlabel
+        # Sick mood special greeting flow
+        if persistent._mas_mood_sick:
+            $ pushEvent("greeting_sick")
```

Comment:
remember that greetings need to be first out, so this should be setting the selected_greeting instead of pushing it here.

Reply:
Sure thing, I changed it to how it must be now. I don't know why I did the pushEvent, I guess I was more worried about it working like it used to and forgot about the new flow. I'll make sure to double check the next time.